tests/ui/error-emitter/multiline-removal-suggestion.rs

```diff
@@ -0,0 +1,53 @@
+// Make sure suggestion for removal of a span that covers multiple lines is properly highlighted.
+//@ compile-flags: --error-format=human --color=always
+//@ edition:2018
+// ignore-tidy-tab
+use std::collections::{HashMap, HashSet};
+fn foo() -> Vec<(bool, HashSet<u8>)> {
+	let mut hm = HashMap::<bool, Vec<HashSet<u8>>>::new();
```

Comment:
Question: is there a specific reason why the indents in this file use tabs?

Reply:
To ensure that the code normalization logic still kicks in correctly, given that I'm doing length calculations involving the original code. `\t` gets always replaced with `   `.

Reply:
Could you add that as a comment in the test? I imagine someone else who looked at this test might notice the tabs and go "huh?" because other ui tests typically only use spaces for indent purposes.